ansible_collections/arista/avd/roles/eos_validate_state/python_modules/tests/avdtestrouting.py

```diff
@@ -101,13 +105,21 @@ def add_test(self, afi: str, bgp_neighbor_ip: str, bgp_peer: str, safi: str | No
             },
         )
 
-    def create_tests(self, afi: str, safi: str | None = None) -> None:
+    def create_tests(
+        self,
+        afi: str,
+        description: str,
+        safi: str | None = None,
+    ) -> None:
         """Create BGP tests for the given AFI and SAFI."""
         bgp_neighbors = get(self.structured_config, "router_bgp.neighbors", [])
+        formatted_afi = afi.replace("-", "_")
+        if safi == "sr-te":
+            formatted_afi += "_sr_te"
```

Comment:
move this in the BGP_ADDRESS_FAMILIES dict using `avd_key` or whatever as the key :)

Reply:
Done